internal/services/containers/kubernetes_fleet_manager_resource.go

```diff
@@ -87,31 +107,34 @@ func (r KubernetesFleetManagerResource) Create() sdk.ResourceFunc {
 	return sdk.ResourceFunc{
 		Timeout: 30 * time.Minute,
 		Func: func(ctx context.Context, metadata sdk.ResourceMetaData) error {
-			client := metadata.Client.ContainerService.V20231015.Fleets
-
-			var config KubernetesFleetManagerResourceSchema
-			if err := metadata.Decode(&config); err != nil {
+			var model KubernetesFleetManagerModel
+			if err := metadata.Decode(&model); err != nil {
 				return fmt.Errorf("decoding: %+v", err)
 			}
 
+			client := metadata.Client.ContainerService.V20231015.Fleets
 			subscriptionId := metadata.Client.Account.SubscriptionId
```

Comment:
Please be mindful of the ordering of calls here, boilerplate code should follow the same pattern and ordering as the rest of the provider.
```suggestion
			client := metadata.Client.ContainerService.V20231015.Fleets
			subscriptionId := metadata.Client.Account.SubscriptionId
			
			var model KubernetesFleetManagerModel
			if err := metadata.Decode(&model); err != nil {
				return fmt.Errorf("decoding: %+v", err)
			}
```